flocker/acceptance/testtools.py

```diff
@@ -28,8 +30,8 @@
 )
 
 from ..control.httpapi import container_configuration_response, REST_API_PORT
-
-from flocker.testtools import loop_until
+from ..cli.script import treq_with_authentication
```

Comment:
I think this import means `treq_with_authentication` needs to go in `flocker.common`.